.gitlab-ci.yml

```diff
@@ -225,8 +237,10 @@ build:aws-k8s-pipeline-toolbox:
   needs: []
   image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:git
   services:
-    - name: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:dind
+    - name: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:dind-rootless
```

Comment:
Remember that these jobs are not run on regular PRs, so to actually test this change in the PR, you need to either protect the branch and run manually.

If it works, we can actually get the rid of the `build:mender-dist-packages-image:manual` and run the jobs normally :clap:

Reply:
Thanks for the head-up!